modules/openapi-generator/src/main/resources/python/python-experimental/api.mustache

```diff
@@ -108,7 +108,7 @@ class {{classname}}(object):
             )
             kwargs['_host_index'] = kwargs.get('_host_index', 0)
 {{#requiredParams}}
-            kwargs['{{paramName}}'] = {{paramName}}
+            kwargs['{{paramName}}'] = {{paramName}}   # noqa: E501
```

Comment:
Can we spread this assignment over a newline and eliminate the `# noqa: E501`?

Reply:
ok... That's what I did initially, but then I noticed developers seem to be using # noqa: E501

Reply:
Yup, I made that call in the past also when I wasn't as focused on the style guide. Recently and going forward if we can pass flake8 checks then we should. If adding the newline still throws E501 line too long, then let's keep it.

Reply:
making change.